src/FPIndia/View.hs

```diff
@@ -53,12 +53,21 @@ renderBody rp model r = do
         renderMarkdown model "resources.md"
     renderFooter
 
-renderNavbar :: Prism' FilePath Route -> H.Html
-renderNavbar rp =
+renderNavbar :: Prism' FilePath Route -> HtmlRoute -> H.Html
+renderNavbar rp currentRoute =
   H.nav ! A.class_ "w-full h-1/4 text-xl font-bold flex space-x-4  mb-4" $ do
-    forM_ universe $ \r -> renderURL (H.toHtml $ routeTitle r) (routeHref rp r)
+    forM_ universe $ \r ->
+      if r == currentRoute
+        then renderURL (H.toHtml $ routeTitle r) (routeHref rp r) True
+        else renderURL (H.toHtml $ routeTitle r) (routeHref rp r) False
```

Comment:
```haskell
renderURL (H.toHtml $ routeTitle r) (routeHref rp r) $ r == currentRoute
```

We don't even need the `H.toHtml ` there if you do it in the function itself.

Reply:
can actually remove routeHref too I believe. Will change